conf/distro/openxt-main.conf

```diff
@@ -57,7 +57,7 @@ PREFERRED_PROVIDER_console-tools = "console-tools"
 PREFERRED_PROVIDER_virtual/libx11 = "libx11"
 
 # Force specific package versions
-PREFERRED_VERSION_networkmanager = "0.9.8.%"
+PREFERRED_VERSION_networkmanager = "1.4.4.%"
```

Comment:
`networkmanager` version is `1.4.4`, this will not match, it should be `1.4.4` or `1.4.%` if the goal is to follow micro releases of the `1.4` line.
```
preferred version 1.4.4.% of networkmanager not available (for item networkmanager)
```

Reply:
Yes, I will change it to `1.4.4` as I think our next goal would be to upgrade against zeus. Thanks